R/attribute-utils.R

```diff
@@ -167,7 +167,8 @@ attr_names.data.frame <- function(x){
 #' @importFrom foreign read.dbf
 #' @export
 #' @keywords internal
-attr_names.shapefile <- function(filename){
+attr_names.shapefile <- function(x){
+  filename <- x
```

Comment:
What does this change accomplish?

Reply:
That was a pkg check warning because the signature had different arg name than generic

Reply:
ohhh, got it